src/routes/webhooks/customerio.ts

```diff
@@ -96,10 +97,21 @@ async function trackCioEvent(payload: ReportingEvent['Body']): Promise<void> {
   await sendAnalyticsEvent([event]);
 }
 
+const validateNotificationPayload = (payload: NotificationPayload): boolean => {
+  return (
+    payload &&
+    payload.notification &&
+    payload.notification.title &&
```

Comment:
`payload?.notification?.title` use optional chaining?

Reply:
Added lots of `?` now :)